_pages/en/strategy-version-control-systems.md

```diff
@@ -0,0 +1,346 @@
+4. Make their projects in the VCS open by default.
+
+## Coherent set of actions
+
+The following are actions that need to be performed in order to make the Version Control Systems operational.
+
+<table>
+  <tr>
+    <th>Outcome</th>
+    <th>Action</th>
+    <th>Description</th>
+    <th>Lead</th>
+    <th>Contributor(s)</th>
+  </tr>
+  <tr>
+    <td rowspan="3"><b>Architecture</b></td>
+    <td>Update ESDC OSS Management Framework</td>
+    <td>Update ESDC OSS Management Framework to reflect the expected usage and capabilities of each approved VCS</td>
+    <td><b>EA</b></td>
+    <td>
+    DevCoP<br>
+    IT Strategy<br>
+    ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td>Finalize endorsement of ESDC OSS Management Framework</td>
+    <td>Endorse the implementation of the framework</td>
+    <td><b>EA</b></td>
+    <td>
+    Senior Mgmt<br>
+    ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td>Define best practices and standards</td>
+    <td>Define best practices, standards and procedures for using Git-based version control systems</td>
+    <td><b>DevCoP</b></td>
+    <td>
+      EA<br>
+      ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="3"><b>Operations</b></td>
+    <td>GCcode:<br>
+    Provide official support</td>
+    <td>Determine the most appropriate model for providing official support in GCcode with the current owner (SSC)</td>
+    <td><b>Software Design Services (SDS)</b></td>
+    <td>
+      SSC<br>
+      TBS<br>
+      DevCoP<br>
+      EA<br>
+      IT Strategy<br>
+      ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td>GCcode:<br>
+    Organize projects and monitor</td>
+    <td>Organize ESDC projects in GCcode and monitor usage</td>
+    <td><b>Software Design Services (SDS)</b></td>
+    <td>
+      ?
+    </td>
+  </tr>
+  <tr>
+    <td>GCcode:<br>
+    Enable connectivity to the Internet</td>
+    <td>Enable projects in GCcode to deploy to public cloud service providers</td>
+    <td><b>SSC</b></td>
+    <td>
+      Cloud CoE<br>
+      ?
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="2"><b>Development</b></td>
+    <td>GCcode:<br>
+    Enable DevOps capabilities</td>
+    <td>
+    Ensure DevOps capabilities are available in GCcode, such as:
+    <ul>
+      <li>Continuous Integration (CI)</li>
+      <li>Continuous Deployment (CD)</li>
+    </ul>
+    </td>
+    <td><b>SSC</b></td>
+    <td>
+    DevCoP<br>
+    ?
+    </td>
+  </tr>
+  <tr>
+    <td>Migrate projects to GCcode</td>
+    <td>Migrate software-related projects, scripts and system/database configurations to the chosen VCS, and gradually sunset legacy version control systems
+    </td>
+    <td><b>Development teams</b></td>
+    <td>
+    DevCoP<br>
+    Operations teams<br>
+    ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="1"><b>Communications</b></td>
+    <td>Train staff</td>
+    <td>Offer training resources to IITB staff related to DevOps and VCS</td>
+    <td><b>Learning Branch</b></td>
+    <td>
+    DevCoP<br>
+    ?
+    </td>
+  </tr>  
+</table>
```

Comment:
I'm thinking an action is going to be needed to tackle older languages and projects: how do we bring COBOL and other projects ran in the past related to mainframes for example? I know it's possible, IBM and subject matter experts have been writing about that but that's partly Tech Debt as well: 
- Leverage refactoring of the code to decouple the monolithic systems we had and that we need to adapt to BDM for example?

Reply:
I agree, this is very much needed. However, I think it goes a bit beyond the scope of this Strategy. These projects would still be able to leverage the VCSs in "legacy fashion". The refactoring should be a Tech Debt Strategy on its own in my view.